entrypoint/createcluster/createcluster_test.go

```diff
@@ -11,6 +11,8 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 )
 
+var arch = customflag.ServiceFlag.ClusterConfig.Arch.String()
```

Comment:
lets follow the same approach in all suites...if you want to use this as a var, create it on the suite file, not in the test it self....lets try at max keep this file clean as possible it can be.

Reply:
please also add this on the other tests places that uses it....to stabilish the same pattern.